src/App.js

```diff
@@ -1,25 +1,82 @@
+
+import React, { useState, useEffect } from 'react';
+import axios from 'axios'
 import './App.css';
```

Comment:
Move styling import to after all the imports to line 12

Reply:
Fixed it.